ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-main.yaml

```diff
@@ -1,15 +1,36 @@
+base_images:
+  ocp_builder_rhel-9-base-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-base-openshift-4.19
```

Comment:
This PR looks good to me. I just wanted to highlight the recent changes in lws-operator about removal of OCP version from the base image tag.

* https://github.com/openshift/release/pull/61493
* https://github.com/openshift/lws-operator/pull/11

There is no functional issue. But conceptually base image is independent from OCP version.